### DISCUSSION TITLE
还原论 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#478] [General]
# 还原论 - Miao Yu | 于淼

https://yufree.cn/cn/2024/10/14/reductionism/

**Comment by zocozocozoco** [comment]
「可解释」看起来也是一种信仰，只不过宗教提供不可解释的解释，科学提供更为自洽的体系，但并不完全自洽。

**Comment by Wfield** [comment]
爱因斯坦说上帝不掷骰子，他应该是认为宇宙可以用规律解释。因为爱因斯坦比我聪明千百倍，所以我跟随他的想法。
